app/components/UserAttendance/AttendanceStatus.css

```diff
@@ -13,13 +13,13 @@
 .poolBox {
   display: flex;
   flex: 1;
-  background-color: rgba(200, 200, 200, 30%);
   max-width: 100%;
   min-width: 25%;
   flex-direction: column;
   align-items: center;
+  background-color: rgba(200, 200, 200, 30%);
```

Comment:
Seems unnecessary, but sure

Reply:
I prefer to have visual stylings that don't affect size/position/etc. at the bottom (color, background, font, border).